<launch>
	<arg name="gui" default="false"/>
	<arg name="world_file" default="$(find turtlebot_rplidar)/worlds/laberint.world"/>
	<arg name="move_base" default="true"/>

	<!--GMAPPING & GAZEBO-->
	<include file="$(find turtlebot_rplidar)/launch/gmapping_gazebo.launch">
		<arg name="gui" value="$(arg gui)"/>
		<arg name="world_file" value="$(arg world_file)"/>
		<arg name="move_base" value="$(arg move_base)"/>
	</include>
	
	<!--RANDOM EXPLORATION-->
	<node pkg="exploracio" 
  		  type="explora_random_node" 
  		  name="explora_random_node" 
  	      output="log">
	</node>
	
	<node name="rviz"
          pkg="rviz"
          type="rviz" 
          args="-d $(find exploracio)/rviz/fronteres.rviz" />
	
</launch>
